# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../faac/faac-1.28-strcasestr.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- faac-1.28/common/mp4v2/mpeg4ip.h.orig	2010-07-23 13:04:17.468255124 +0200
+++ faac-1.28/common/mp4v2/mpeg4ip.h	2010-07-23 13:05:46.756753305 +0200
@@ -123,7 +123,7 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-char *strcasestr(const char *haystack, const char *needle);
+//char *strcasestr(const char *haystack, const char *needle);
 #ifdef __cplusplus
 }
 #endif
